doc/OnlineDocs/explanation/philosophy/expressions/managing.rst

```diff
@@ -216,19 +202,20 @@ callbacks, which are documented in the class documentation.
 Detailed documentation of the APIs for these methods is provided
 with the class documentation for these visitors.
 
-SimpleExpressionVisitor Example
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+StreamBasedExpressionVisitor Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In this example, we describe an visitor class that counts the number
 of nodes in an expression (including leaf nodes).  Consider the following
 class:
 
 .. literalinclude:: /src/expr/managing_visitor1.spy
```

Comment:
This code snippet still references `SimpleExpressionVisitor` and should be updated: https://github.com/jsiirola/pyomo/blob/identify-vars/doc/OnlineDocs/src/expr/managing.py#L104